snakemake_for_automation.md

```diff
@@ -31,38 +31,18 @@ Install `snakemake` using conda.
 conda install -y -c bioconda snakemake-minimal
 ```
 
-Type the following in your terminal to display a link to Rstudio web-server for your instance’s $(hostname)
+Type the following in your terminal to display a link to Rstudio web-server 
+for your instance’s $(hostname)
 
 ```
 echo http://$(hostname):8787/
```

Comment:
I keep making these $(hostname -i), because that uses the IP address that we are using for ssh login. Introduces one less concept, and keeps one less strange ID being thrown around.  

e.g. `echo http://$(hostname -i):8787/` works the same but with the IP address we're using

Reply:
ah clever. ok. again, will change once we decide what interface we're using.

Reply:
lgtm